flags.go

```diff
@@ -372,14 +401,18 @@ func populateFlags(c *cli.Context) (flags *flagStorage) {
 		OpRateLimitHz:                      c.Float64("limit-ops-per-sec"),
 
 		// Tuning,
-		MaxRetrySleep:     c.Duration("max-retry-sleep"),
-		StatCacheCapacity: c.Int("stat-cache-capacity"),
-		StatCacheTTL:      c.Duration("stat-cache-ttl"),
-		TypeCacheTTL:      c.Duration("type-cache-ttl"),
-		LocalFileCache:    c.Bool("experimental-local-file-cache"),
-		TempDir:           c.String("temp-dir"),
-		DisableHTTP2:      c.Bool("disable-http2"),
-		MaxConnsPerHost:   c.Int("max-conns-per-host"),
+		MaxRetrySleep:          c.Duration("max-retry-sleep"),
+		StatCacheCapacity:      c.Int("stat-cache-capacity"),
+		StatCacheTTL:           c.Duration("stat-cache-ttl"),
+		TypeCacheTTL:           c.Duration("type-cache-ttl"),
+		LocalFileCache:         c.Bool("experimental-local-file-cache"),
+		TempDir:                c.String("temp-dir"),
+		DisableHTTP2:           c.Bool("disable-http2"),
+		MaxConnsPerHost:        c.Int("max-conns-per-host"),
+		MaxIdleConnsPerHost:    c.Int("max-idle-conns-per-host"),
+		DisableKeepAlives:      c.Bool("disable-keep-alives"),
+		ForceAttemptHTTP2:      c.Bool("force-attempt-http2"),
+		EnableHTTP1ForGoClient: c.Bool("enable_http1_for_go_client"),
```

Comment:
we are adding too many flags, why not reuse DisableHTTP2 flag in go client library also.